services/provider/server/server.go

```diff
@@ -418,6 +426,21 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	//Enable CSI-OMAP is storageClusterPeer is present
+	clientMappingConfig, err := s.getClientMapping(ctx)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Same as last comment

Reply:
This function is failing in cases expect isNotFound